.travis.yml

```diff
@@ -22,33 +22,41 @@ before_script:
 
 jobs:
   include:
-    - stage: Verify
+    - stage: Build
       script:
         - make install.tools
         - make .gitvalidation
         - make verify
+        - make binaries
       go: 1.8.x
     - script:
         - make install.tools
         - make .gitvalidation
         - make verify
+        - make binaries
       go: tip
-    - stage: Build and Test
+    - stage: Test
       script:
         - make install.deps
         - make test
-        - make binaries
```

Comment:
mostly just pointing out that deleting the -make binaries line did not remove the call to make binaries.  Yeah an extra build is acceptable since it's so fast. Just pointing out it's also unnecessary on the 1.8 build because you'll be doing it twice... not sure why it's confusing to have a build in a verify step but not verify in a build step :-)